docs/docs/noir/standard_library/meta/module.md

```diff
@@ -8,6 +8,12 @@ declarations in the source program.
 
 ## Methods
 
+### add_item
+
+#include_code add_item noir_stdlib/src/meta/module.nr rust
+
+Adds a top-level item to the module.
```

Comment:
Can you give some examples here for users that may not know what a top-level item is? Functions, structs, trait impls